Resources/Private/Plugins/Kitodo/Templates/PageView/Main.html

```diff
@@ -24,121 +24,14 @@
-                                <f:cObject typoscriptObjectPath="plugin.tx_dlf_imagemanipulationtool" />
-                            </div>
-
-                            <div class="tx-dlf-navigation-next">
-                                <f:if condition="{docPage.{iterator.index}} < {docNumPage.{iterator.index}}">
-                                    <f:then>
-                                        <f:link.action title="{f:translate(key: 'nextPage')}" class="next" addQueryString="untrusted" additionalParams="{navigationArray.{iterator.index}.next}" argumentsToBeExcludedFromQueryString="{0: 'tx_dlf[docMeasure][{iterator.index}]'}">
-                                            <f:translate key="nextPage" />
-                                        </f:link.action>
-                                    </f:then>
-                                    <f:else>
-                                        <span class="next" title="{f:translate(key: 'nextPage')}">
-                                            <f:translate key="nextPage" />
-                                        </span>
-                                    </f:else>
-                                </f:if>
-
-                                <f:link.action class="nextMeasure"title="{f:translate(key: 'nextMeasure')}" addQueryString="untrusted" additionalParams="{navigationMeasureArray.{iterator.index}.next}" argumentsToBeExcludedFromQueryString="{0: 'tx_dlf[docPage][{iterator.index}]'}">
-                                    <f:translate key="nextMeasure" />
-                                </f:link.action>
-                            </div>
-
-                        </div>
-                        <div class="gridstack-dragging-handle" title="{f:translate(key: 'multiview.dragging_label', extensionName: 'dfgviewer')}"></div>
-                        <f:variable name="multipleSourceValue" value="{kitodo:multipleSource(docArrayKey: docArrayKey)}" />
-                        <f:if condition="{multipleSourceValue} >= 0">
-                            <div class="removeDocument">
-                                <f:link.action addQueryString="untrusted" argumentsToBeExcludedFromQueryString="{0: 'tx_dlf[multipleSource][{multipleSourceValue}]'}">
-                                    <f:translate key="multiview.remove_document" extensionName="dfgviewer" />
-                                </f:link.action>
-                            </div>
-                        </f:if>
-                    </div>
-                </f:for>
-            </div>
-        </div>
-    </f:then>
-    <f:else>
-        <div id="tx-dfgviewer-map-0" class="tx-dlf-map"></div>
-        <div id="svg_panel">
-            <f:asset.script identifier="wildwebmidi-remote-package-base" priority="true">
-                const WILDWEBMIDI_REMOTE_PACKAGE_BASE = '{viewData.publicResourcePath}/JavaScript/WildWebMidi/wildwebmidi.data';
-            </f:asset.script>
-            <div id="player"></div>
```

Comment:
Where do I find now the player?

Reply:
The player isn’t working properly, and finding a maintained alternative is still on my to-do list. So for now, it’s been temporarily removed from this part.